### PR TITLE
Next fixes

### PR DESCRIPTION
**Pull request by homeworkprod**
Hey Alice, I finally managed to get a deeper go at the `next` branch (on Python 3.9, to which I just upgraded).

I came across a few errors right away for which I have provided what I think should be fixes (unless I misinterpreted your intention behind the code pieces in question).

Let me know what you think. :)

### CODE REVIEW

**Comment by amcgregor** [comment]
I'll have a gander at this shortly. Apologies for the insane delay.

File: marrow/mailer/address.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def __init__(self, addresses=None, encoding="utf-8"):
 	
 	def __repr__(self):
 		if not self: return "AddressList()"
-		return f"AddressList(\"{", ".join([str(i) for i in self])}\")")
+		return f"AddressList(\"{', '.join(str(i) for i in self)}\")"
```
Comment (amcgregor) [inline marrow/mailer/address.py:146]: A ha, no, the original comprehension was actually correct! 😁 Good catch on the missing parenthesis, though; that's my bad. What you've replaced it with is a list comprehension, which, while still functional, stops and then creates a concrete list from the result of a generator comprehension adding the memory overhead of a true list to each evaluation. The original was the bare [generator expression](https://docs.python.org/3.9/glossary.html#index-20). Join accepts ["iterable objects"](https://docs.python.org/3/library/stdtypes.html#str.join), which both lists and generators qualify as.

**Review by amcgregor — DISMISSED** [review]
~I only request that you revert the generator/list comprehension change, otherwise~ thank you so very much for contributing! Once again, my deepest apologies for taking so long to get to this. It's been… a crazy time, shall we say.

File: marrow/mailer/address.py (hunk 1 above)
Comment (homeworkprod) [reply marrow/mailer/address.py:146]: Hey,

> What you've replaced it with is a list comprehension

are you sure? I believe it *was* a list comprehension `[… for …]`, and I have removed the square brackets to turn it into a more efficient generator expression.

**Comment by homeworkprod** [comment]
It seems to me you've applied pretty much these changes (including the LC → GE change) in 46b1c20a6672d51b9a9cf008aa4a581b1928903d already anyway.

File: marrow/mailer/address.py (hunk 1 above)
Comment (amcgregor) [reply marrow/mailer/address.py:146]: Yup, nope, my dyslexia got the best of me here. 🙇 Apologies. Sometimes, to go faster, I need to slow down.

**Comment by amcgregor** [comment]
Closing as already implemented. Temporary variable use instead of just switching which quotes were in use (d'oh!) but still effective.

**Pull request closed without merging**